base/unicode/utf8.jl

```diff
@@ -26,10 +26,6 @@ const utf8_trailing = [
     2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2, 3,3,3,3,3,3,3,3,4,4,4,4,5,5,5,5,
 ]
 
-# Retained because although undocumented and unexported, used in a package (MutableStrings)
```

Comment:
A PR or issue should probably be opened on MutableStrings since this package is mentioned explicitly.